nautobot/extras/tests/integration/test_jobs.py

```diff
@@ -82,14 +85,16 @@ def get_cell_value(row, column):
         # Test for log level (one row should be visible)
         filter_element.fill("")
         filter_element.type(log_entries[3].log_level.title())
+        time.sleep(1) # Wait for API call to return a response
```

Comment:
Instead of using sleep are you able to use splinter's [`is_element_present`](https://splinter.readthedocs.io/en/stable/matchers.html#matchers) with a `wait_time` kwarg?